gradle-plugins/compose/src/test/test-projects/misc/commonResources/expected-open-res/commonMainResourceCollectors/my/lib/res/ExpectResourceCollectors.kt

```diff
@@ -0,0 +1,25 @@
+package my.lib.res
+
+import kotlin.String
+import kotlin.collections.Map
+import org.jetbrains.compose.resources.DrawableResource
+import org.jetbrains.compose.resources.ExperimentalResourceApi
+import org.jetbrains.compose.resources.FontResource
+import org.jetbrains.compose.resources.PluralStringResource
+import org.jetbrains.compose.resources.StringArrayResource
+import org.jetbrains.compose.resources.StringResource
+
+@ExperimentalResourceApi
+public expect val Res.allDrawableResources: Map<String, DrawableResource>
```

Comment:
Just something to consider: providing a `Map` is a large API surface. It's good for the user, but could be hard for us to provide/maintain. If we just provide a getter function instead, it would limit the API surface to the bare minimum.

Reply:
> I would drop the all prefix, but it's really a minor nit.

then IDE will suggest you
```
Res.drawab_ (Ctrl+Space)
    drawable //object for accessors
    drawableResources //map with all drawables
```

I find it confusing